src/mds/mdstypes.cc

```diff
@@ -532,7 +532,7 @@ void session_info_t::dump(Formatter *f) const
   f->open_array_section("prealloc_inos");
   for (const auto& [start, len] : prealloc_inos) {
```

Comment:
start is not range here.

Reply:
Can you clarify what you mean? I'm not sure what you're saying here.

Reply:
never mind.